tests/base.py

```diff
@@ -632,15 +635,11 @@ def getPRHeadSha(self):
         repo = self._getRepo()
         return repo.references[self._getPRReference()].commit.hexsha
 
-    def setStatus(self, state, url, description, context):
-        self.statuses[context] = {
-            'state': state,
-            'url': url,
-            'description': description
-        }
+    def setStatus(self, context, state):
+        self.statuses.append('%s:%s' % (context, state))
```

Comment:
You need to update the assert at tests/test_github.py:235 with the new status format